docs/preview/03-Features/logging-azure-functions.md

```diff
@@ -80,4 +82,84 @@ Next, make sure that you pass along the exception middleware to the exception ha
 {
     app.UseExceptionHandling<MyExceptionHandlingMiddleware>();
 });
+```
+
+## Logging incoming requests
+The `AzureFunctionsRequestTrackingMiddleware` class can be added to the Azure Functions worker pipeline to log any incoming HTTP requests.
+The requests are tracked with [Arcus Observability](https://observability.arcus-azure.net/Features/writing-different-telemetry-types#incoming-http-requests-in-azure-function-http-trigger) so that they will show up as requests in Application Insights, when the application is using [Arcus Application Insights Serilog sink](https://observability.arcus-azure.net/Features/sinks/azure-application-insights).
+
+> ⚠ The HTTP request and response body are not tracked by default.
+
+> ⚡ The HTTP request headers are logged by default, except certain security headers are by default omitted: `Authentication`, `X-Api-Key` and `X-ARR-ClientCert`.
+
+Tracking a HTTP request will look like this in the logs:
+`HTTP Request POST http://localhost:5000/weatherforecast completed with 200 in 00:00:00.0191554 at 03/23/2020 10:12:55 +00:00 - (Context: [Content-Type, application/json], [Body, {"today":"cloudy"}])`
+
+### Usage
+To use this middleware component, add the following line to your startup code:
+```csharp
+using Microsoft.Extensions.Hosting;
+
+public class Program
+{
+    public static void Main(string[] args)
+    {
+        IHost host = new HostBuilder()
+            .ConfigureFunctionsWorkerDefaults(builder =>
+            {
+                builder.UseRequestTracking();
+            })
+            .Build();
+    
+        host.Run();
+    }
+}
+```
+
+### Configuration
+The middleware component can be configured to influence the behavior of the HTTP request tracking. This ranges from including HTTP request/response bodies, specifying which routes should be tracked, filtering HTTP request headers, and more.
+To learn more about these options, see [the configuration section at the general Web API page](./logging.md) as these options are identical for Azure Functions HTTP triggers and Web API's.
+
+### Customization
+Optionally, the middleware component can be extended even further by inheriting from the `AzureFunctionsRequestTrackingMiddleware` class. This allows full control over the sanitation process of the HTTP request/response body and HTTP request headers.
```

Comment:
```suggestion
Optionally, the middleware component can be extended even further by inheriting from the `AzureFunctionsRequestTrackingMiddleware` class. This allows full control over the sanitization process of the HTTP request/response body and HTTP request headers.
```

Reply:
Hihi, was probably not adding this in vscode where I have a spell check. Thx!